modules/virt-copying-the-ssh-command.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-copying-the-ssh-command_{context}"]
+= Copying the SSH Command from the web console
+
+Copy the SSH command of your virtual machine (VM) from the web console for use in the OpenShift CLI (`oc`).
+
+.Prerequisites
+* A configured virtual machine.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+. Click the *Virtual Machines* tab.
+. Select a virtual machine to open the *Virtual Machine Overview* page.
+. Click the *Actions* drop-down menu and select *Copy SSH Command*.
```

Comment:
Per the IBM style guide, refer to a drop-down list by its label, followed by "list".

Suggestion:
````
From the *Actions* list, select *Copy SSH Command*.
````